docker-compose-updater/Dockerfile

```diff
@@ -1,21 +1,28 @@
 # Extends the official python Dockerimage by adding the docker-compose-updater
 #
-#  Author: Tobias Malkmus
-#  E-Mail: malkmus@mps-med.de
-#
 # This Dockerfile adheres to the best practices for Dockerfiles please consider
 # this when modifying the file
 # https://docs.docker.com/develop/develop-images/dockerfile_best-practices/
 
-FROM python:3.6-alpine
+FROM python:3.8.2-alpine3.11
 
 WORKDIR /
 
-# TODO: improve caching here
-COPY --chown=1000:1000 requirements.txt /
-RUN pip install -r ./requirements.txt && rm /requirements.txt
-COPY --chown=1000:1000 src/docker_compose_update.py /docker_compose_update.py
+# Install Docker and docker-compose
+RUN apk add \
+docker \
+gcc \
+libc-dev \
+libffi-dev \
+make \
+openssl-dev \
+python-dev \
+&& pip install docker-compose
```

Comment:
Sehe ich das gerade nicht oder wofür brauchen wir Docker in dem Container ?

Reply:
Das Script führt docker-compose up aus, docker-compose führt Docker aus. Um mit dem Docker Socket reden zu können braucht man Docker.

Reply:
Da sehe ich mal wieder den Wald vor lauter Bäumen nicht ;)